Updating the Linkerd description (#13058)
DataDog/integrations-core: linkerd/README.md

## Overview

> Linkerd][1] is an ultra-light, ultra-simple, ultra-powerful open-source service mesh that provides developers with the tools they need to write secure, reliable, observable cloud-native applications. It is the only service mesh to reach CNCF graduated status. Linkerd uses mutual TLS to transparently secure all on-cluster TCP communication with no application changes, instantly adds latency-aware load balancing, request retries, timeouts, and blue-green deploys to keep your applications resilient, and provides instant platform health metrics by tracking success rates, latencies, and request volumes for every meshed workload — all with minimal configuration.

Reviewer: ```suggestion
[Linkerd][1] is an ultra-light, ultra-simple, ultra-powerful open-source service mesh that provides developers with the tools they need to write secure, reliable, observable cloud-native applications. It is the only service mesh to reach CNCF graduated status. Linkerd uses mutual TLS to transparently secure all on-cluster TCP communication with no application changes, instantly adds latency-aware load balancing, request retries, timeouts, and blue-green deploys to keep your applications resilient, and provides instant platform health metrics by tracking success rates, latencies, and request volumes for every meshed workload - all with minimal configuration.
```

Reviewer: I'm wondering if you can split the third sentence into other sentences or a bulleted list?

> Linkerd][1] is an ultra-light, ultra-simple, ultra-powerful open-source service mesh that provides developers with the tools they need to write secure, reliable, observable cloud-native applications. It is the only service mesh to reach CNCF graduated status. Linkerd uses mutual TLS to transparently secure all on-cluster TCP communication with no application changes, instantly adds latency-aware load balancing, request retries, timeouts, and blue-green deploys to keep your applications resilient, and provides instant platform health metrics by tracking success rates, latencies, and request volumes for every meshed workload — all with minimal configuration.

Reviewer 2: ```suggestion
[Linkerd][1] is a light but powerful open-source service mesh with CNCF graduated status. It provides the tools you need to write secure, reliable, observable cloud-native applications. With minimal configuration and no application changes, Linkerd:

- Uses mutual TLS to transparently secure all on-cluster TCP communication. 
- Adds latency-aware load balancing, request retries, timeouts, and blue-green deploys to keep your applications resilient.
- Provides platform health metrics by tracking success rates, latencies, and request volumes for every meshed workload.
```

Author: Oops, I think I committed the wrong suggestion. Copied and pasted this one in the text, though. Thanks!

> This integration allows Datadog users to send their Linkerd metrics, including application success rates, latency, and saturation to their Datadog dashboards. The Datadog agent will collect metrics from the Linkerd proxy and make them available in your Datadog account.

Reviewer 2: ```suggestion
This integration sends your Linkerd metrics to Datadog, including application success rates, latency, and saturation.
```
